mne/coreg.py

```diff
@@ -1756,6 +1756,23 @@ def _setup_icp(self, n_scale_params):
             mri_pts *= self._scale  # not done in fit_matched_points
         return head_pts, mri_pts, weights
 
+    def set_fid_match(self, match):
+        """Set the strategy for fitting anatomical landmark (fiducial) points.
+
+        Parameters
+        ----------
+        match : 'nearest' | 'matched'
+            Alignment strategy; ``'nearest'`` aligns anatomical landmarks to
+            any point on the head surface; ``'matched'`` aligns to the fiducial
+            points in the MRI.
+        """
+        if match not in self._icp_fid_matches:
+            msg = (f'"match" must be one of {self._icp_fid_matches}, got '
+                   f'{match}')
+            raise ValueError(msg)
```

Comment:
Should be
```suggestion
        _check_option('match', match, self._icp_fid_matches)
```
But also needs an import at the top